app/uk/gov/hmrc/tariffclassificationfrontend/audit/AuditService.scala

```diff
@@ -60,6 +60,14 @@ class AuditService @Inject()(auditConnector: DefaultAuditConnector) {
     )
   }
 
+  def auditRulingCancelled(oldCase: Case, updatedCase: Case, operator: Operator)
```

Comment:
cancel case or cancel ruling?

Reply:
I believe its canceling the ruling - at least in the ticket and the prototype UI.  I've tried to be consistent with this in the code.